stubs/gdb/gdb/__init__.pyi

```diff
@@ -289,7 +288,14 @@ class PendingFrame:
 class UnwindInfo:
     def add_saved_register(self, reg: str | RegisterDescriptor | int, value: Value, /) -> None: ...
 
-frame_unwinders: list[gdb.unwinder.Unwinder]
+class _Unwinder(Protocol):
```

Comment:
I assume this doesn't exist at runtime. Could you add a `@type_check_only` decorator (needs import from `typing`)?

```suggestion
@type_check_only
class _Unwinder(Protocol):
```

Reply:
Done.